README.md

```diff
@@ -4,47 +4,86 @@
 Placeholder for the Metadata Query Service
 
 
+# Query Service
+[![Test Coverage](https://codecov.io/gh/HumanCellAtlas/query-service/branch/master/graph/badge.svg)](https://codecov.io/gh/HumanCellAtlas/query-service)
+[![Master Deployment Status](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/badges/master/pipeline.svg)](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/commits/master)
+
+######TODO add these links
+[![Production Health Check](health check badge SVG)](URL to service)
```

Comment:
```suggestion
[![Production Health Check](health check badge SVG)](https://status.data.humancellatlas.org/service/query-service-prod.svg)
```

Reply:
This will display `service | unknown` until you deploy. The URL should be the same.

Reply:
thanks!